parse.go

```diff
@@ -228,6 +229,14 @@ func AllowMismatchPixelDataLength() ParseOption {
 	}
 }
 
+// AllowErrorMetaElementGroupLength allows parser to work around missing metaelement group length tag (0x0002,0x0000) by reading elements only
+// in group 2
+func AllowErrorMetaElementGroupLength() ParseOption {
```

Comment:
would this read better as `AllowMissingMetaElementGroupLength`?

Reply:
Agree, changed the name.